README.md

```diff
@@ -69,9 +69,10 @@ Dependencies
 
 ABySS requires the following libraries:
 
-* [Boost](http://www.boost.org)
-* [sparsehash](http://code.google.com/p/sparsehash)
+* [Boost](http://www.boost.org/)
 * [Open MPI](http://www.open-mpi.org)
+* [sparsehash](https://code.google.com/p/sparsehash/)
+* [SQLite](http://www.sqlite.org/)
```

Comment:
This dependency is optional. In fact, Boost is the only required dependency, but the others are heavily recommended. Should we also mention a compiler capable of OpenMP here?